src/atomics/multithread.md

```diff
@@ -0,0 +1,220 @@
+# Multithreaded Execution
+
+The first important thing to understand about C++20 atomics is that **the
+abstract machine has no concept of time**. You might expect there to be a single
```

Comment:
Good to front it with this, but it might also be worth a couple small paragraphs differentiating the abstract machine from the physical machine, and perhaps motivate why we care to make that distinction.